analysis.tf

```diff
@@ -0,0 +1,136 @@
+resource "aws_instance" "analysis" {
+  key_name                    = "${var.key_name}"
+  ami                         = "${data.aws_ami.rhel_67.id}"
+  instance_type               = "r4.4xlarge"
+  iam_instance_profile        = "${aws_iam_instance_profile.analysis.id}"
+  vpc_security_group_ids      = ["${aws_security_group.analysis.id}"]
+  associate_public_ip_address = false
+  subnet_id                   = "${aws_subnet.ops_public_subnet.id}"
+  private_ip                  = "${var.analysis_instance_ip}"
+
+  user_data = <<EOF
+EOF
+
+  tags = {
+    Name = "ec2-analysis-${local.naming_suffix}"
+  }
+
+  lifecycle {
+    prevent_destroy = false
+  }
+}
+
+
+
+
+
+resource "aws_security_group" "analysis" {
+  vpc_id = "${aws_vpc.opsvpc.id}"
+
+  ingress {
+    from_port = 443
+    to_port   = 443
+    protocol  = "tcp"
+
+    cidr_blocks = ["${var.analysis_cidr_ingress}"]
+  }
+
+  ingress {
+    from_port = 22
+    to_port   = 22
+    protocol  = "tcp"
+
+    cidr_blocks = [
+      "${var.ad_sg_cidr_ingress}",
+    ]
+  }
+
+  egress {
+    from_port = 0
+    protocol  = "-1"
+    to_port   = 0
+
+    cidr_blocks = [
+      "0.0.0.0/0",
+    ]
+  }
+
+  tags {
+    Name = "sg-analysis-${local.naming_suffix}"
+  }
+}
+
+
+
+
+
+resource "aws_eip" "analysis_eip" {
```

Comment:
This module is not in use

Reply:
forgot to attach to the instanced - fixed